plugins/sslservices/sslservices.cpp

```diff
@@ -700,15 +681,15 @@ OPENSSL_API void OPENSSL_CALL pkRSAUnseal(ICodeContext *ctx, size32_t & __lenRes
                 failOpenSSLError("EVP_OpenInit");
 
             // Allocate memory for the plaintext
-            int plaintextLen = newCipherTextLen;
+            size32_t plaintextLen = newCipherTextLen;
             plaintext.ensureCapacity(plaintextLen);
 
-            int len = 0;
-            if (EVP_OpenUpdate(decryptCtx, static_cast<byte *>(plaintext.bufferBase()), &len, newCipherText, newCipherTextLen) != 1)
+            size32_t len = 0;
```

Comment:
same comment about int being correct for this call/instance

Reply:
Fixed.